src/app/content/highlights/components/Highlights.spec.tsx

```diff
@@ -400,9 +406,16 @@ describe('Highlights', () => {
       const [firstDeleteWrapper, secondDeleteWrapper] = component.root.findAllByType(HighlightDeleteWrapper);
       firstDeleteWrapper.props.onDelete();
       secondDeleteWrapper.props.onCancel();
+
+      requestRemoveHighlight.hookBody({...services, getState: store.getState, dispatch: store.dispatch})(
+        requestDeleteHighlight(hlBlue as Highlight, {
+          locationFilterId: pageId,
+          pageId,
+        }
+      ));
```

Comment:
Why does it need to be called manually?

Reply:
it seems like a middleware with hooks is not passed in `createTestStore`
It kind of makes sense, right?